events/start_handler.go

```diff
@@ -22,6 +22,7 @@ const RancherSystemLabelKey = "io.rancher.container.system"
 const RancherIPEnvKey = "RANCHER_IP="
 const RancherNameserver = "169.254.169.250"
 const RancherDomain = "rancher.internal"
+const RancherDns = "io.rancher.dns"
```

Comment:
`io.rancher.network.dns` Sounds good to me.... Is there a table of all the special labels and values yet to compare against?